pwiz_aux/msrc/utility/vendor_api/Waters/WatersRawFile.hpp

```diff
@@ -524,6 +538,26 @@ struct PWIZ_API_DECL RawData
         }
     }
 
+    private:
+
+	void readAnalogChromatograms()

```

Comment:
This is a stray tab instead of space indentation.

I do want to see performance measurement for this because it's reading all the analog data just to open the file even if the user isn't interested in the analog data.

Reply:
Done.

Reply:
Performance measurement still needs to be done. I´m working on it.